src/applications/static-pages/homepage-hero-randomizer/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import './sass/homepage-hero.scss';
+
+// shuffle array using Fisher-Yates shuffle
+const randomizeOrder = imageArray => {
+  const tempArray = imageArray;
+  let currentIndex = Object.keys(tempArray).length;
```

Comment:
Shouldn't need `Object.keys...` here. `tempArray.length` should suffice.  If we want to expand this function to be usable by more than just